stl/inc/algorithm

```diff
@@ -9228,8 +9289,29 @@ namespace ranges {
 #endif // __cpp_lib_concepts
 #endif // _HAS_CXX17
 
+template <class _FwdIt, class _Pr>
+_INLINE_VAR constexpr bool _Is_min_max_optimization_safe =
+    _Iterator_is_contiguous<_FwdIt>&&
+        conjunction_v<disjunction<is_integral<_Iter_value_t<_FwdIt>>, is_pointer<_Iter_value_t<_FwdIt>>>,
+            disjunction<is_same<_Pr, less<>>, is_same<_Pr, _RANGES less>,
+                is_same<_Pr, less<_Iter_value_t<_FwdIt>>>>> && !is_volatile_v<remove_reference_t<_Iter_ref_t<_FwdIt>>>;
+
 template <class _FwdIt, class _Pr>
 constexpr _FwdIt _Max_element_unchecked(_FwdIt _First, _FwdIt _Last, _Pr _Pred) { // find largest element
+#if _USE_STD_VECTOR_ALGORITHMS
+    if (!is_constant_evaluated()) {
+        if constexpr (_Is_min_max_optimization_safe<_FwdIt, _Pr>) {
+            const auto _First_ptr = _To_address(_First);

```

Comment:
Missing `_STD`

Reply:
In every pre-existing `_To_address` use

Reply:
It is related to #140

Reply:
I think @miscco might have originally been referring to `if (!is_constant_evaluated()) {` which was indeed missing `_STD`. That code has changed, and you're correct that we don't currently qualify `_Ugly` functions, so resolving this comment.